app/src/main/java/com/example/musicapp/ui/main/MainViewModel.kt

```diff
@@ -10,8 +10,7 @@ class MainViewModel: ViewModel() {
     var instruments: MutableLiveData<ArrayList<Instrument>> = MutableLiveData<ArrayList<Instrument>>()
     var instrumentService: InstrumentService = InstrumentService()
 
-    fun getAllInstruments(){
+    internal fun getAllInstruments(){
```

Comment:
This is another example of when you should use the internal visibility modifier so that this function is only available in the module.